tests/PHPUnit/Framework/TestCase/SystemTestCase.php

```diff
@@ -617,6 +619,10 @@ public function assertDbConnectionCreated($message = 'A database connection was
         self::assertTrue(Db::hasDatabaseObject(), $message);
     }
 
+    public static function provideContainerConfigBeforeClass()
+    {
+        return array();
+    }
```

Comment:
This one is static (used by static method), the other is instance (should have access to test instance to set/use properties). Can't have the same name otherwise PHP will complain.